fmn/consumer/backends/base.py

```diff
@@ -1,7 +1,19 @@
 import abc
-
 import logging
+import requests
 import fmn.lib.models
+log = logging.getLogger("fedmsg")
+
+
+def shorten(link):
```

Comment:
moved the shorten url method from irc backend to base so that multiple backends can utilize the function